chains/evm/listener/listener.go

```diff
@@ -75,10 +77,14 @@ func (l *EVMListener) ListenToEvents(ctx context.Context, startBlock *big.Int, m
 		case <-ctx.Done():
 			return
 		default:
+			ctxWithSpan, span := otel.Tracer("relayer-core").Start(ctx, "relayer.core.EVMListener.ListenToEvents")
```

Comment:
Won't this create a span for each iteration?
It seems like too much maybe.
I think logging is fine in terms of getting to handling deposits and thne from there tracing takes over.

Reply:
It creates a span at least every 5 sec, and this is the most high-level function in our listener, I do not think this will overload relayer to much. And this for example already gave me some insights that blockRetryInterval for example might be at least three times longer

Reply:
I think I set it to block time on prod, but it seems reasonable